tests/test_yummly.py

```diff
@@ -12,32 +12,49 @@ def test_host(self):
 
     def test_canonical_url(self):
         self.assertEqual(
-            "https://www.yummly.com/recipe/Carrot-Milk-shake-1099424",
+            "https://www.yummly.com/recipe/Easy-White-Cheese-_-Garlic-Pizzas-9351327",
             self.harvester_class.canonical_url(),
         )
 
     def test_title(self):
-        self.assertEqual(self.harvester_class.title(), "Carrot Milk shake")
+        self.assertEqual(
+            self.harvester_class.title(), "Easy White Cheese & Garlic Pizzas"
+        )
 
     def test_total_time(self):
-        self.assertEqual(30, self.harvester_class.total_time())
+        self.assertEqual(25, self.harvester_class.total_time())
 
     def test_yields(self):
         self.assertEqual("4 serving(s)", self.harvester_class.yields())
 
     def test_ingredients(self):
+
+        print(">>>>", self.harvester_class.ingredients())
```

Comment:
NB: Debug print for removal here.  Is this part of development/testing process you've been using?

I wonder if we can find a way to make that easier.  From memory, it can certainly be tricky to extract the diffs from the test suite output when a test fails (even with verbose and `maxDiff` settings applied)

Reply:
fixed. Sorry about that

Reply:
No worries, thanks!

Reply:
If there's a canonical logger implementation we could tie into that so that it shows different log levels.